app/controllers/admin/mentors_controller.rb

```diff
@@ -1,4 +1,18 @@
 class Admin::MentorsController < ApplicationController
+
   def index
+    @user_ids = TrackMentorship.distinct.pluck(:user_id)
```

Comment:
The only variables used in the view are `@users` and `@tracks` so the rest don't need to be instant variables (ie, they don't need the `@`).